.github/workflows/bump-elastic-stack-snapshot.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Install Updatecli in the runner
-        uses: updatecli/updatecli-action@b0d4fd84ae72108930c9adb83bb4beba1acbaf9f # v2.79.0
+        uses: updatecli/updatecli-action@339fdb3f3bf66d14fe5830248b4e893990415cd0 # v2.80.0
```

Comment:
If possible, let's avoid the bump, I'm working on fixing some of the existing issues with the current approach for Beats, and eventually here too.

See https://github.com/elastic/beats/pull/43743

Reply:
Okay, I can revert the `updatecli/updatecli-action` version bump in this PR.  Just to double check, there is no problem with keeping the `golangci/golangci-lint-action` version bump in this PR, correct, @v1v?

Reply:
Yep, correct, thanks @ycombinator

Reply:
Thanks, done.